src/main/java/com/aws/greengrass/util/exceptions/ProcessCreationException.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.util.exceptions;
+
+import java.io.IOException;
+
+public class ProcessCreationException extends IOException {
```

Comment:
Exec caller code already handles IOException. Planning to use this exception to wrap some runtime exception so that the error can be handled by callers